mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping

```diff
@@ -50,22 +50,22 @@ CLASS net/minecraft/class_1293 net/minecraft/entity/effect/StatusEffectInstance
 		ARG 1 that
 	METHOD equals (Ljava/lang/Object;)Z
 		ARG 1 o
-	METHOD method_24275 fromTag (Lnet/minecraft/class_1291;Lnet/minecraft/class_2487;)Lnet/minecraft/class_1293;
+	METHOD method_24275 fromNbt (Lnet/minecraft/class_1291;Lnet/minecraft/class_2487;)Lnet/minecraft/class_1293;
 		ARG 0 type
-		ARG 1 tag
+		ARG 1 nbt
 	METHOD method_24276 copyFrom (Lnet/minecraft/class_1293;)V
 		ARG 1 that
-	METHOD method_24277 typelessToTag (Lnet/minecraft/class_2487;)V
-		ARG 1 tag
+	METHOD method_24277 saveTypelessToNbt (Lnet/minecraft/class_2487;)V
```

Comment:
agreed, but again not my name, just reformatted the existing name. I don't think typeless is a good description of what it does but I'm not sure what would be better. This method basically adds all the info of a status effect such as amplifier and all that stuff but without a status effect type.

Reply:
This is to prevent writing the extraneous status effect type info to the hidden effect tag. Feel free to come up with better names like `saveInfoToNbt`

Reply:
I went with savePropertiesToNbt since that's a bit more descriptive than info.